wrappers/csharp/src/Native.cs

```diff
@@ -12,8 +15,11 @@ namespace Devolutions.Cryptography
     public static partial class Native
     {
 #if RDM
-        private const string LibName64 = "x64/DevolutionsCrypto";
-        private const string LibName86 = "x86/DevolutionsCrypto";
+        [DllImport("Kernel32", SetLastError = true)]
+        public static extern IntPtr LoadLibrary(string path);
+
+        private const string LibName64 = "DevolutionsCrypto";
+        private const string LibName86 = "DevolutionsCrypto";
 #endif
```

Comment:
@thenextman  I remember that we were required to have the native libraries in the x86 and x64 folder in the RDM environment is it still a thing?  (That is why we have 2 nugets for windows)

Reply:
@MathieuMorrissette No; x86 is no longer shipped in RDM and could be removed

We can simplify this a lot if I can remove the x86 completely (the bindings are all written twice for x86 and x64 but the signatures are not different)

Are the x86 bindings used anywhere else? I know Android still builds x86 for example.

Reply:
@thenextman I think x86 is still used in some powershell modules. What I meant is that we have 2 nugets package. 

- Devolutions.Crypto.Windows.RDM.nupkg (different package because of special native library folder requirement)
    - Only used by RDM windows and related components.
    - Drops the native library in the Debug/x64 folder (required by RDM team)
        - All native libraries must be located in this folder. 
- Devolutions.Crypto.Windows.nupkg
    - Used by other windows projects that are not related to RDM. 
    - Nuspec : https://github.com/Devolutions/devolutions-crypto/blob/master/wrappers/csharp/nuget/Windows/Devolutions.Crypto.Windows.nuspec

Reply:
@MathieuMorrissette Thanks that clears things up. We can remove the "special" RDM package for Windows, since the libraries will be consolidated in the same place for all platforms.

Maybe I can pull the `LoadLibrary` stuff up into RDM as well (.NET isn't smart enough to load the proper native library from the runtimes folder until .NET 6)

How is this handled on other platforms?

- It looks like iOS and Android you only ship managed code?
- For macOS you have "Full" (native code) and "Modern" (managed only). Is the "Full" package used? Does RDM choose which library to load (`dlopen`)?
- Same for Linux - how is RDM choosing which native library to load? Or it "just works" because RDM Linux is .NET 6 (I think)

Do you know any of these questions?

Reply:
Yeah I would remove the special "RDM" package if the special requirement is no longer required.

- Android : The native library is embedded in the managed DLL. (It's the way Xamarin Android generate nuget package)
- macOS: I don't know if the "Full" is still used. The native library is also embedded is the "Modern" package. (see the native reference in the macOS nuget project https://github.com/Devolutions/devolutions-crypto/blob/master/wrappers/csharp/nuget/macOS/Modern/Devolutions.Crypto.Mac/Devolutions.Crypto.Mac/Devolutions.Crypto.Mac.csproj#L47)
- Linux : It just works. even if it isn't .net 6, it still looks for native library in the runtime folder. (I'm not sure how or why it works)

Reply:
Thanks @MathieuMorrissette. I will revise the PR.